### DISCUSSION TITLE
Documentation correction by users

### DISCUSSION THREAD

**Discussion by Pewibe** [arendst/Tasmota#11696] [General]
Is it possible to correct and improve documentation by ordinary users? If yes, tell me
- how?
- if I'm not sure in writing a behavior - can I mark suggestion or alternatives that an insider will prove and finalize it?
- does somebody of the insiders have to check my input anyway?

**Comment by barbudor** [comment]
> How

Click on the pencil icon on top right

> if I'm not sure in writing a behavior - can I mark suggestion or alternatives that an insider will prove and finalize it?
> does somebody of the insiders have to check my input anyway?

Your proposed change will be publish as a PR (Pull Request) just like any code change
In addition of the changes you will write an little introduction to explain your changes and why
The change request will be reviewed and we may ask you to change things either in the syntax or in the text
You change will be only merged if/when they fulfill the standard.

**Comment by TrDA-hab** [comment]
https://tasmota.github.io/docs/Contributing/

**Reply by Pewibe** [reply]
@TrDA-hab  Just for my personal understanding: Do I have overseen this good written information or have you (or someone else) added it since @barbudor answered my question?

**Reply by barbudor** [reply]
The answer is in Github
https://github.com/tasmota/docs/blame/development/docs/Contributing.md
You will se who and when this was added

**Reply by TrDA-hab** [reply]
Honestly, it really works (but very strange), the information update rate is usually from 0.5 hour to 30 days (I still don't understand how it works) ... He didn't wait any longer.
If you don't like to wait (like me) you can leave an instruction here https://github.com/arendst/Tasmota/discussions in the category: "Show and tell" (it works instantly and doesn't depend on anyone). 
Later (possibly) the moderators will transfer your instructions (or make a link to it in WIKI Tasmota). In any case, you should understand that this is not a commercial project and the moderators work using their personal time (no taking time for the family) ... I am very grateful to them !!! Good luck !!!
